SRC/element/TclElementCommands.cpp

```diff
@@ -151,7 +151,7 @@ extern void *OPS_ElastomericBearingBoucWenMod3d(void);
 extern void *OPS_PFEMElement2DBubble(const ID &info);
 extern void *OPS_PFEMElement2Dmini(const ID &info);
 extern void *OPS_PFEMElement2D();
-#ifdef _HAVE_LHNMYS
+#if defined(_HAVE_LHNMYS) || defined(OPSDEF_ELEMENT_LHNMYS)
```

Comment:
Added alternate standard compliant macro name for enabling Prof. Filippou's LHNMYS element.